Dangerfile

```diff
@@ -0,0 +1,13 @@
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = github.pr_title.include? "#trivial"
+
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
```

Comment:
Yo, what if you use a regex o match this tag? There might have developers that use [wip] or [WIP] or even wip without square brackets.